llm-server/routes/root_service.py

```diff
@@ -107,7 +219,17 @@ def extract_data(data: Dict[str, Any]) -> Tuple:
     headers = data.get("headers", {})
     server_base_url = cast(str, data.get("server_base_url", ""))
     app = headers.get("X-App-Name") or None
-    return text, swagger_url, session_id, base_prompt, headers, server_base_url, app
+    bot_id = data.get("bot_id", None)
+    return (
+        text,
+        swagger_url,
+        session_id,
+        base_prompt,
+        headers,
+        server_base_url,
+        app,
+        bot_id,
```

Comment:
@gharbat expecting `bot_id` here. This will create a namespace in qdrant with bot_id, so each user's knowledbase is private.